app/models/mhv_account.rb

```diff
@@ -99,6 +100,10 @@ def user
     @user ||= User.find(user_uuid)
   end
 
+  def mhv_correlation_id
+    user&.mhv_correlation_id
```

Comment:
probably don't want to have `&.` here if you're saying && in the clause that uses this.

Reply:
The safe navigator was used before inline with the `&&` expression, though. Was that also incorrect?

Reply:
if mhv_correlation_id is only used here, and not elsewhere, maybe it's worth just removing it and using `user.mhv_correlation_id`

but `user&.mhv_correlation_id && <something>` doesn't make sense unless user can be nil

Reply:
It's being used in two places, which is why I introduced the method.

Reply:
I'll remove the safe navigation operator anyway, because MhvAccount only gets instantiated from a User, so `user` couldn't be nil here.

Reply:
I didn't think it was that necessary before, but kept it since that's how it was written previously.